### DISCUSSION TITLE
WSS support

### DISCUSSION THREAD

**Discussion by charliez0** [vercel/vercel#5560] [Help]
It seems that vercel don't support wss despite docs https://vercel.com/docs/edge-network/overview?query=wss#supported-protocols. I want to use vercel as a websocket proxy to external site pubsub.codeforces.com
Config are shown:
```json
"rewrites": [{
    "source": "/pubsub.codeforces.com/ws/(.*)",
    "destination": "wss://pubsub.codeforces.com/ws/$1"
  }]
```
However vercel seems to ifnore this rule. If I change wss:// to https:// it works but websocket connection failed of course.

**Comment by popsee** [comment]
The same demand i need, but i can't find an effective config or method to handle it. 
Wss proxy didn't work.
Anyone could tell me how to deal with it, looking forword to your good news!

**Comment by fyang93** [comment]
same here, need support to websocket
